docs/src/catalyst_applications/bifurcation_diagrams.md

```diff
@@ -1,115 +1,118 @@
+BifurcationKit computes bifurcation diagrams using the `bifurcationdiagram` function. From an initial point in the diagram, it tracks the solution (using a continuation algorithm) until the entire diagram is computed (BifurcationKit's continuation can be used for other purposes, however, this tutorial focuses on bifurcation diagram computation). The continuation settings are provided in a `ContinuationPar` structure. In this example, we will only specify three settings, `p_min` and `p_max` (which sets the minimum and maximum values over which the bifurcation parameter is varied) and `max_steps` (the maximum number of continuation steps to take as the bifurcation diagram is tracked). We wish to compute a bifurcation diagram over the interval *(2.0,20.0)*, and will use the following settings:
 ```@example ex1
-p_bstart = copy(p)
-p_bstart[bif_par] = p_span[1]
-u0 = [:X => 1.0, :A => 1.0]
-nothing   # hide
+p_span = (2.0, 20.0)
+opts_br = ContinuationPar(p_min = p_span[1], p_max = p_span[2], max_steps=1000)
+nothing # hide
 ```
-Finally, we extract the ODE derivative function and its jacobian in a form that
-BifurcationKit can use:
+
+Finally, we compute our bifurcation diagram using:
 ```@example ex1
-oprob = ODEProblem(rn, u0, (0.0, 0.0), p_bstart; jac = true)
-F = (u,p) -> oprob.f(u, p, 0)
-J = (u,p) -> oprob.f.jac(u, p, 0)
-nothing   # hide
+bif_dia = bifurcationdiagram(bprob, PALC(), 2, (args...) -> opts_br; bothside=true)
+nothing # hide
 ```
+Where `PALC()` designates that we wish to use the pseudo arclength continuation method to track our solution. The third argument (`2`) designates the maximum number of recursions when branches of branches are computed (branches appear as continuation encounter certain bifurcation points). For diagrams with highly branches structure (rare for most standard chemical reaction networks) this input is important. Finally, `bothside=true` designates that we wish to perform continuation on both sides of the initial point (which is typically the case). 
 
-In creating an `ODEProblem` an ordering is chosen for the initial condition and
-parameters, and regular `Float64` vectors of their numerical values are created
-as `oprob.u0` and `oprob.p` respectively. BifurcationKit needs to know the index
-in `oprob.p` of our bifurcation parameter, `:S`, and the index in `oprob.u0` of
-the variable we wish to plot, `:X`. We calculate these as
+We can plot our bifurcation diagram using the Plots.jl package:
 ```@example ex1
-# get S and X as symbolic variables
-@unpack S, X = rn
-
-# find their indices in oprob.p and oprob.u0 respectively
-bif_idx  = findfirst(isequal(S), parameters(rn))
-plot_idx = findfirst(isequal(X), species(rn))
-nothing   # hide
+using Plots
+plot(bif_dia; xguide="k1", yguide="X")
 ```
+Here, the steady state concentration of *X* is shown as a function of *k1*'s value. Stable steady states are shown with thick lines, unstable ones with thin lines. The two fold bifurcation points are marked with "bp".
```

Comment:
Can we make "fold bifurcation point" a reference to a webpage that describes type of bifurcations?

Reply:
https://en.wikipedia.org/wiki/Saddle-node_bifurcation